lib/plugin.js

```diff
@@ -24,6 +24,11 @@ ManifestPlugin.prototype.getFileType = function(str) {
   return ext;
 };
 
+ManifestPlugin.prototype.existNameInManifest = function(name, manifest) {
```

Comment:
It is not enough to test whether we have the entry in manifest keys, values need checking too. 

As far i see, we have assets in [chunk.files](https://github.com/danethurber/webpack-manifest-plugin/blob/v1.1.2/lib/plugin.js#L50) and in [stats.assets](https://github.com/danethurber/webpack-manifest-plugin/blob/v1.1.2/lib/plugin.js#L66) storing processed ones in `moduleAssets` and `manifest`, in `moduleAssets` we receive nothing, which was initial motivation of this PR, then in `manifest` i tested first only keys for avoiding duplication, it results we need more.

When using hashed values in assets name, key testing is not enough, I need to check whether file name is not in value also. For example for tests using hashes and base path, i got manifests like:
```
{ 'one.1067da9bcb5b3a357a83.js': 'one.1067da9bcb5b3a357a83.js',
  'one.js': 'one.1067da9bcb5b3a357a83.js',
  'third.party.js': 'third.party.js' }
```
the first entry in that object is wrong.